src/main/java/app/bpartners/api/endpoint/event/model/UserNonSubscribed.java

```diff
@@ -0,0 +1,31 @@
+package app.bpartners.api.endpoint.event.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Duration;
+import lombok.*;
+
+@Builder
+@ToString
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserNonSubscribed extends PojaEvent {
```

Comment:
No.

The event must refer to the action that triggered the event not the goal.  

Here it could be UserRegistrationRequested for example.